recipes/sicer2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "SICER2" %}
+{% set version = "1.0.1" %}
+{% set pypi_hash = "5d/ea/4da9fde475d3a4b236f0c359f275c369333799d3519de9ce0b0fbaf5714c" %}
+{% set github = "https://github.com/zanglab/SICER2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/{{ pypi_hash }}/{{ name }}-{{ version }}.tar.gz"
```

Comment:
You can replace this with
```
url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
and then you don’t need the `pypi_hash` variable.

Reply:
Thanks, switched this